### ISSUE TITLE
App launcher role attributes

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-3#1063]
The roles currently applied to the menu in the [app launcher](https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/application-launcher.html) are incorrect. 

 `<li>` should have `role=“presentation”` 
 `<a>` should have `role=“menuitem”`

[explanation about these roles](https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-links.html)
